py/moduerrno.c

```diff
@@ -102,20 +102,6 @@ const mp_obj_module_t mp_module_uerrno = {
 };
 
 const char* mp_errno_to_str(mp_obj_t errno_val) {
```

Comment:
should this call mp_common_errno_to_str()? or remove gutted function so it doesnt get called by accident?

Reply:
It can't because we only want to allocate to the stack. decompressing here would require a heap allocation since the stack disappears on return. I think we do really want to call both.

Reply:
np - plz grep to check all calls to this are now calling _common_ :)

Reply:
Done!

```
/V/E/circuitpython (huffman|✔) $ ag -Q mp_errno_to_str .
py/moduerrno.c
104:const char* mp_errno_to_str(mp_obj_t errno_val) {
127:const char* mp_errno_to_str(mp_obj_t errno_val) {

py/mperrno.h
143:const char* mp_errno_to_str(mp_obj_t errno_val);

py/objexcept.c
124:                    msg = mp_errno_to_str(o->args->items[0]);
```